src/__tests__/process.test.js

```diff
@@ -34,16 +43,25 @@ describe('createProcess', () => {
     expect(spawn.mock.calls[0][1]).toEqual([]);
   });
 
-  it('spawns a command with spaces from workspace.pathToJest', () => {
-    const workspace: any = {pathToJest: 'npm test --'};
+  it('spawns the command from workspace.jestCommandLine ignoring workspace.pathToJest', () => {
+    const workspace: any = {jestCommandLine: 'jest', pathToJest: 'not jest'};
```

Comment:
this is not a valid workspace, we should not have `pathToJest` property in workspace any more...